### ISSUE TITLE
Object doesn't support property or method 'find' (Internet Explorer 11)

### ISSUE THREAD

**Issue by nparley** [mapbox/mapbox-gl-draw#575]
Using my own code and the example https://www.mapbox.com/mapbox-gl-js/example/mapbox-gl-draw/ I am getting a bug when using internet explorer 11. The message says:

```
Object doesn't support property or method 'find'
File: mapbox-gl-draw.js, Line: 2, Column: 18798
```
which is this bit of code here:
```
{"style"===t.dataType&&!function(){var t=e.setup,n=e.map,r=e.options,o=e.store,i=!!r.styles.find(function(e){return n.getLayer(e.id)});i||(t.addLayers(),o.setDirty(),o.render())}()};var d={trash:!1,combineFeatures:!1,uncombineFeatures:!1},h={changeMode:t,actionable:n,currentModeName:function()
```

all ok on chrome

**Comment by davidtheclark** [comment]
[IE11 does not support `Array.find`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/find#Browser_compatibility)! What kind of world is this.

**Comment by davidtheclark** [comment]
A fix will be released momentarily in 0.16.1.

**Comment by nparley** [comment]
Thanks 👍
